samples/buildstrategy/ko/buildstrategy_ko_cr.yaml

```
@@ -69,31 +88,54 @@ spec:
             fi
           done
 
+          # Determine the ko version
+          KO_VERSION='$(params.ko-version)'
+          if [ "${KO_VERSION}" == "latest" ]; then
+            KO_VERSION=$(curl --silent "https://api.github.com/repos/google/ko/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+          fi
+
+          # Create one variable with v-suffix and one without as we need both for the download URL
+          if [[ ${KO_VERSION} = v* ]]; then
+            KO_VERSION_WITH_V=${KO_VERSION}
+            KO_VERSION_WITHOUT_V=${KO_VERSION:1}
+          else
+            KO_VERSION_WITH_V=v${KO_VERSION}
+            KO_VERSION_WITHOUT_V=${KO_VERSION}
+          fi
+
           # Download ko
           pushd /tmp > /dev/null
-            curl -f -s -L https://github.com/google/ko/releases/download/v0.8.3/ko_0.8.3_$(uname)_$(uname -m | sed 's/aarch64/arm64/').tar.gz | tar xzf - ko
+            curl -f -s -L "https://github.com/google/ko/releases/download/${KO_VERSION_WITH_V}/ko_${KO_VERSION_WITHOUT_V}_$(uname)_$(uname -m | sed 's/aarch64/arm64/').tar.gz" | tar xzf - ko
```

We can simplify this by removing the `pushd`/`popd` with `-C` flag of `tar`, which does the change directory.
```suggestion
            curl -f -s -L "https://github.com/google/ko/releases/download/${KO_VERSION_WITH_V}/ko_${KO_VERSION_WITHOUT_V}_$(uname)_$(uname -m | sed 's/aarch64/arm64/').tar.gz" | tar xzf - -C /tmp ko
```

---

Related to that, I wonder why not install it into `/usr/local/bin`, this way we can simplify the calls later by just calling `ko`.

---

Nice, thank you. :-) Changed it to use the `-C` flag. We cannot install into `/usr/local/bin` because the strategy runs as non-root.